configure.ac

```diff
@@ -283,6 +283,11 @@ case "${host}" in
 	abi="elf"
 	AC_DEFINE([JEMALLOC_PURGE_MADVISE_FREE], [ ])
 	;;
+  *-*-bitrig*)
+	CFLAGS="$CFLAGS"
+	abi="elf"
+	AC_DEFINE([JEMALLOC_PURGE_MADVISE_FREE], [ ])
+	;;
   *-*-openbsd*)
```

Comment:
I think you should join bitrig case with openbsd case to avoid both code duplication and readers' confusion about what is "bitrig"
```
*-*-openbsd*|*-*-bitrig*)
  ...